index.html

```diff
@@ -28,6 +27,132 @@ <h1>Hemerson Foreste</h1>
           <button id="introduction-button" type="button">Start collaboration</button>
           <img src="images/image_geometry_2.svg" alt="image">
       </section>
+
+          <!--Projects Section-->
+          <section id="projects">
```

Comment:
- [ ]  In the end, this website will be responsive so kindly change it to `display: grid`.  to this section
Check-in [here](https://www.geeksforgeeks.org/comparison-between-css-grid-css-flexbox/#:~:text=CSS%20Grids%20helps,the%20same%20time.) why you should use grid.
[Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20in%20%22Works%22%20section%20you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both).%20Use%20CSS%20Grid%20in%20the%20main%20container%20that%20contains%20all%20of%20the%20cards%20and%20use%20Flexbox%20to%20lay%20out%20an%20individual%20card.).